internal/server/kong/ws/config/configuration.go

```diff
@@ -50,11 +52,33 @@ func (l *KongConfigurationLoader) Register(mutator Mutator) error {
 func (l *KongConfigurationLoader) Load(ctx context.Context, clusterID string) (Content, error) {
 	var configTable DataPlaneConfig = map[string]interface{}{}
 	for _, m := range l.mutators {
+		mutationStartTime := time.Now()
 		err := m.Mutate(ctx, MutatorOpts{ClusterID: clusterID},
 			configTable)
+		mutationDuration := time.Since(mutationStartTime).Milliseconds()
 		if err != nil {
+			metrics.Histogram("data_plane_config_mutation_time_individual",
```

Comment:
Can we follow standard Prometheus conventions: 

1. Naming this metric series: `data_plane_config_mutation_individual_duration_seconds`
    - We're not emitting the entire mutation process as a metric. Do we want to be doing that too?
2. Altering to push seconds instead (with a decimals)

Reply:
1. [This is tracks total](https://github.com/Kong/koko/pull/485/files#diff-4b5bf8ad94aaa24b01cab08505b8ac550fc4efe48eac16233df472395212c06dR481), which accounts for success and fail states as well
2. I take this to mean `t=.001s` vs `t=1ms`. I'm accustomed to emitting ms, is emitting decimal seconds a prometheus convention?

Reply:
1. Yes, that's fine.
2. Derp, I forgot to link the doc. Yes, it's a convention. https://prometheus.io/docs/practices/naming/